### PR TITLE
Fix `fetch-providers.sh` script for Terraform >= 0.13

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area usability open-source
/kind bug
/priority 3

**What this PR does / why we need it**:
Beginning with Terraform 0.13, plugins must be installed in a file system with the following hierarchy:
`registry.terraform.io/<username>/<plugin-name>/<version>/<arch>`

❗ Please note that this will PR is not enough to enable updating a Terraform version from < 0.13 to >= 0.13. It only works well for new providers (like `equinixmetal`).

**Release note**:

```bugfix developer
A bug has been fixed preventing to use Terraformer with a Terraform version >= 0.13.
```

### CODE REVIEW

File: build/fetch-providers.sh [hunk 1]
```diff
@@ -4,14 +4,26 @@
...
+if [ "$(version "$TF_VERSION")" -lt "$(version "0.13.0")" ]; then
+  mkdir "tfproviders"
+  find . -name "terraform-provider*" | xargs -I % cp % ./tfproviders/
+else
+  # Beginning with Terraform 0.13, plugins must be installed in a file system with the following hierarchy:
+  # registry.terraform.io/<username>/<plugin-name>/<version>/<arch>
+  # Above `terraform-bundle` command already maintains such structure, hence, we keep it instead of copying
+  # the `terraform-provider` binaries.
+  find . -name "*.md" | xargs -I % rm -f %
+  mv ./plugins ./tfproviders/
```
Comment (timuthy) [inline build/fetch-providers.sh:28]: Don't we need to create the `tfproviders` dir before, as done in the `if` branch above?

Comment (rfranzke) [reply build/fetch-providers.sh:28]: This line is just renaming the `plugins` dir (already containing the correct file system hierarchy`) to `tfproviders`. Hence, no need to pre-create it.

Comment (timuthy) [reply build/fetch-providers.sh:28]: Okay, thank you. I was confused by the slash at the end.

**Review by timuthy — APPROVED** [review]
/lgtm

**Pull request merged**